src/training/flava_eval/finetune_glue.py

```diff
@@ -122,21 +147,35 @@ def __init__(self, task, split, text_key, label_key, tokenizer=None):
 
         self.dataset = load_dataset("glue", task, split=split)
         self.label_key = label_key
-        self.text_key = text_key
+        self.text_key = text_key # list of strings
+        assert len(self.text_key) <= 2 and isinstance(self.text_key, list)
```

Comment:
should swap the order of the `and`, otherwise it'll crash if it's not an iterable/list